saos-webapp/src/main/resources/saos.default.properties

```diff
@@ -96,4 +96,7 @@ piwik.siteId = 0
 
 
 # ehcache configuration file path
-ehcache.configurationFilePath = classpath:ehcache.xml
\ No newline at end of file
+ehcache.configurationFilePath = classpath:ehcache.xml
+
+# domain name
+domain.name = https://saos.org.pl
```

Comment:
And, the domain name should be taken from request rather than to be const string (the same for every saos instance), see the usage of HttpServletRequestUtils.